src/services/ghost/GhostProvider.ts

```diff
@@ -72,6 +75,12 @@ export class GhostProvider {
 	}
 
 	public async promptCodeSuggestion() {
+		this.taskId = crypto.randomUUID()
+
+		TelemetryService.instance.captureEvent(TelemetryEventName.INLINE_ASSIST_QUICK_TASK, {
+			taskId: this.taskId,
```

Comment:
Should we call this something else to prevent confusion with real taskIds?

Reply:
@chrarnoldus I just checked the implementation with Justin and he want to use the same parameter to keep consistent the same structure of this event.

Reply:
It's basically to keep the same structure to work with the same dashboard that they have.